src/smart-components/portfolio/portfolio.js

```diff
@@ -68,9 +68,18 @@ const Portfolio = props => {
     .catch(() => dispatch({ type: 'setCopyInProgress', payload: false }));
   };
 
+  const handleEmptyPage = () => {
+    dispatch({ type: 'setIsFetching', payload: true });
+    return props.fetchPortfolioItemsWithPortfolio(props.match.params.id, defaultSettings)
```

Comment:
We should keep the existing page limit here, and not switch to the default.

Reply:
Right switch to first page but keep the limit :+1: